Packages/CatbirdApp/Makefile

```diff
@@ -0,0 +1,22 @@
+test:
```

Comment:
It is more convenient with Makefile when it is one at the root, so that everything can be run from one place.

Reply:
Done.